interpreter/k8s/src/main/scala/io/buoyant/interpreter/k8s/IstioInterpreter.scala

```diff
@@ -1,54 +1,46 @@
 package io.buoyant.interpreter.k8s
 
 import com.twitter.finagle._
-import com.twitter.finagle.buoyant.PathMatcher
 import com.twitter.finagle.naming.NameInterpreter
 import io.buoyant.k8s.istio.RouteCache
-import io.buoyant.namer.{ConfiguredDtabNamer, RewritingNamer}
+import io.buoyant.namer.ConfiguredDtabNamer
 import istio.proxy.v1.config.RouteRule
 
 object IstioInterpreter {
-  private val noLabelsPfx = "/#/no-labels"
   private val istioPfx = "/#/io.l5d.k8s.istio"
   private val defaultRouteDtab = Dtab.read(s"""
-    |/svc/dest => /$$/inet ;
-    |/destPath/local/cluster/svc => $noLabelsPfx ;
-    |/svc/dest => /$$/io.buoyant.http.domainToPathPfx/destPath ;
+    |/svc/ext => /$$/inet ;
```

Comment:
In the future we may want to send this to the egress linkerd instead of directly to the internet.